src/Symfony/Component/Form/FormConfigBuilderInterface.php

```diff
@@ -161,6 +161,17 @@ public function setRequired($required);
     public function setPropertyPath($propertyPath);
```

Comment:
I think the `setPropertyPath()` method should be deprecated. You also need to add the corresponding `trigger_error()` statements.